src/controllers/TripController.js

```diff
@@ -0,0 +1,69 @@
+import dotenv from 'dotenv';
+import Customize from '../helpers/Customize';
+import emailHelper from '../helpers/EmailHelper';
+import { tripRequest, tripRequestCities } from '../database/models';
+
+
+dotenv.config();
+/**
+ * @export
+ * @class TripController
+ */
+class TripController {
+  /**
+   * User can be able to request one-way trip
+   * @static
+   * @param {object} req request object
+   * @param {object} res response object
+   * @memberof UserController
+   * @returns {object} data
+   */
+  static async requestTrip(req, res) {
+    try {
+      const userId = req.user.id;
+      const {
+        reason, city, startDate, returnDate
+      } = req.body;
+      const locationArr = city.split(',');
+      const newTrip = await tripRequest.create({
+        userId, reason, status: 'pending', startDate, returnDate, typeTrip: 1
```

Comment:
I think you should set this status(`pending`) by default in the migration file of this table `tripRequest`;
for example:    
status: {
        type: Sequelize.STRING,
        defaultValue: 'pending'
      }